src/Sarif.FunctionalTests/ConverterTestData/AndroidStudio/fileEmpty.xml.sarif

```diff
@@ -24,14 +24,14 @@
       },
       "results": [
         {
+          "analysisTarget": {
+            "uri": "northwindtraders-droid-2/src/main/java/com/northwindtraders/droid/model/Model.java",
+            "uriBaseId": "$PROJECT_DIR$"
+          },
```

Comment:
No: If a `location` object has only one `physicalLocation`, populate `location.physicalLocation`, not `result.analysisTarget`.